src/scss/base/_typography.scss

```diff
@@ -143,4 +144,3 @@ h6 {
   color: $text-sub;
   font-size: ms(-2);
 }
-
```

Comment:
Huh, does the current config not want a new line at eof? Or do we have extra lines rn?

Reply:
Most of the files had 2 newlines at the end. The linter did not like that.